tests/files/test-image.php

```diff
@@ -311,11 +311,11 @@ public function test__get_resized_filename() {
 		$postmeta = get_post_meta( $attachment_id, '_wp_attachment_metadata', true );
 
 		$image = new Automattic\VIP\Files\Image( $postmeta, $attachment_post_data['post_mime_type'] );
-		$width = $this->getProperty( 'width' );
+		$width = $this->get_property( 'width' );
 		$width->setValue( $image, 150 );
-		$height = $this->getProperty( 'height' );
+		$height = $this->get_property( 'height' );
 		$height->setValue( $image, 150 );
-		$get_resized_filename = $this->getMethod( 'get_resized_filename' );
+		$get_resized_filename = $this->get_method( 'get_resized_filename' );
```

Comment:
The more robust approach would be to subclass the `Automattic\VIP\Files\Image`, and provide public access to `set_width_height()` and `get_resized_filename()` methods. This would eliminate the need to use the Reflection API and improve the readability of the code (not to mention that it would make static analysis tools happier).

Reply:
That sounds great, we could leave it for another PR.